### PR TITLE
Remove ControlledVocabularies by CSV upload

### LINKED ISSUE #1242: CVs no longer updating
Edited Controlled Vocabularies are not getting updated in Vireo when uploaded in Controlled Vocabulary Management.  Vireo accepts the file and does not notice a change before confirming the upload.  In the example I was working with, I removed one of the names in the vocabulary.

### PR DESCRIPTION
**Pull request by jsavell**
Resolves #1242 

#1253 should be merged before this PR.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <parent>
     <groupId>edu.tamu.weaver</groupId>
     <artifactId>webservice-parent</artifactId>
-    <version>2.0.0</version>
+    <version>2.x-WRO-INIT-SNAPSHOT</version>
```
Comment (jeremythuff) [inline pom.xml]: Is this material to the cv behavior?

File: src/test/resources/application.yaml [hunk 2]
```diff
@@ -135,3 +135,4 @@ theme:
   manager: false
   default.css:
   defaults.location:
+  initialize: false
```
Comment (jeremythuff) [inline src/test/resources/application.yaml]: Is this related to the WRO weaver change?

File: pom.xml (hunk 1 above)
Comment (jsavell) [reply pom.xml]: These changes are from #1253, which should be merged very soon.
